bug: "value is unknown" after update to v1.20.0+

**Describe the bug**

After updating from tfsec v0.19.1 to v1.20.0 or higher, a "value is unknown" error is happening.

**To Reproduce**
Steps to reproduce the behavior:
1. Run tfsec on our Terraform project

**Expected behavior**

Run without an issue like tfsec v0.19.1

**Screenshots/Output**

```
[debug:eval][root] Finished processing 299 submodule(s).
[debug:parse][root] Finished parsing module 'root'.
panic: value is unknown

goroutine 163 [running]:
github.com/zclconf/go-cty/cty.Value.AsString({{{0x13d3c48, 0x400012a0ea}}, {0xec0f40, 0x1ef42a0}})
	/home/runner/go/pkg/mod/github.com/zclconf/go-cty@v1.10.0/cty/value_ops.go:1262 +0x150
github.com/aquasecurity/defsec/pkg/terraform.(*Reference).KeyBracketed(0x400176a5a0)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/terraform/reference.go:147 +0xe4
github.com/aquasecurity/defsec/pkg/terraform.(*Reference).String(0x400176a5a0)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/terraform/reference.go:110 +0x198
github.com/aquasecurity/defsec/internal/types.(*Metadata).ToRego(0x40095e0cc8)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/internal/types/metadata.go:27 +0x6c
github.com/aquasecurity/defsec/pkg/rego/convert.StructToRego({0x103ef20, 0x400b955080, 0x99})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/struct.go:55 +0x620
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0x103ef20, 0x400b955080, 0x99})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:34 +0x3c0
github.com/aquasecurity/defsec/pkg/rego/convert.StructToRego({0x1026020, 0x4010cc8718, 0x199})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/struct.go:41 +0x3d0
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0x1026020, 0x4010cc8718, 0x199})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:34 +0x3c0
github.com/aquasecurity/defsec/pkg/rego/convert.SliceToRego({0xe593c0, 0x400b92fc98, 0x97})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/slice.go:28 +0x2e8
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0xe593c0, 0x400b92fc98, 0x97})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:36 +0x40c
github.com/aquasecurity/defsec/pkg/rego/convert.StructToRego({0x102f620, 0x400ddaff70, 0x199})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/struct.go:41 +0x3d0
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0x102f620, 0x400ddaff70, 0x199})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:34 +0x3c0
github.com/aquasecurity/defsec/pkg/rego/convert.SliceToRego({0xe59380, 0x4006b9ecf8, 0x97})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/slice.go:28 +0x2e8
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0xe59380, 0x4006b9ecf8, 0x97})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:36 +0x40c
github.com/aquasecurity/defsec/pkg/rego/convert.StructToRego({0xf6ce40, 0x4006b9ece0, 0x99})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/struct.go:41 +0x3d0
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0xf6ce40, 0x4006b9ece0, 0x99})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:34 +0x3c0
github.com/aquasecurity/defsec/pkg/rego/convert.StructToRego({0x105e920, 0x4006b9e700, 0x99})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/struct.go:41 +0x3d0
github.com/aquasecurity/defsec/pkg/rego/convert.anonymousToRego({0x105e920, 0x4006b9e700, 0x99})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/anonymous.go:34 +0x3c0
github.com/aquasecurity/defsec/pkg/rego/convert.StructToRego({0xebbba0, 0x4001a91c00, 0x16})
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/rego/convert/struct.go:41 +0x3d0
github.com/aquasecurity/defsec/pkg/state.(*State).ToRego(0x4001a91c00)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/state/state.go:31 +0xdc
github.com/aquasecurity/defsec/pkg/scanners/terraform/executor.(*regoJob).Run(0x400e47d8e0)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/scanners/terraform/executor/pool.go:162 +0x3c
github.com/aquasecurity/defsec/pkg/scanners/terraform/executor.(*Worker).Start.func1(0x40009b2f98, 0x40048d52c0)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/scanners/terraform/executor/pool.go:281 +0x34
github.com/aquasecurity/defsec/pkg/scanners/terraform/executor.(*Worker).Start(0x40048d52c0)
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/scanners/terraform/executor/pool.go:286 +0xac
created by github.com/aquasecurity/defsec/pkg/scanners/terraform/executor.(*Pool).Run
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.53.2/pkg/scanners/terraform/executor/pool.go:55 +0xc4

```
**System Info**
 - tfsec version: `v1.20.2`
 - terraform version: `v1.1.9`
 - OS: `macOS 12` & `Docker Container`

**Example Code**

Code base is quite big; if we can figure out which module is the source of problem, I can share the code for sure.

**Additional context**

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.4
 	github.com/Masterminds/semver v1.5.0
-	github.com/aquasecurity/defsec v0.60.0
+	github.com/aquasecurity/defsec v0.60.1
 	github.com/hashicorp/go-version v1.5.0
 	github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
 	github.com/liamg/clinch v1.6.1

